### PR TITLE
opensearch/logs: update integration content

### PR DESCRIPTION
**Pull request by laysauchoa**
Update information in how to manage logs
integration in OpenSearch service

# What changed, and why it matters

### CODE REVIEW

File: docs/products/opensearch/howto/opensearch-log-integration.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Manage OpenSearch log integration
-=================================
+Manage OpenSearch logs integration
```
Comment (anelook) [inline docs/products/opensearch/howto/opensearch-log-integration.rst:1]: I think it should be kept as it is as "log integration". When I search for titles with similar names across other websites, it is common to keep "log" single.

@TibsAtWork , any opinion on this?

File: docs/products/opensearch/howto/opensearch-log-integration.rst [hunk 2]
```diff
@@ -12,23 +12,33 @@ Here are the steps needed to enable logs integration. This allows you to send lo
...
 
-4. Choose an existing OpenSearch instance or create a new one, then select **Continue**.
+4. Select an existing OpenSearch instance or create a new one, then select **Continue**.
 
-5. Configure your ``index prefix`` and ``retention limit`` parameters, then select **Enable**
+5. Configure your ``index prefix`` and ``index retention limit`` parameters, then select **Enable**
 
 .. note::
-    If you want to effectively disable the index ``retention limit``, you can set it to the maximum value which is 10000 days.
+    If you want to effectively disable the index ``index retention limit``, you can set it to the maximum value which is 10000 days.
```
Comment (anelook) [inline docs/products/opensearch/howto/opensearch-log-integration.rst:24]:
```suggestion
    If you want to effectively disable the ``index retention limit``, you can set it to the maximum value which is 10000 days.
```

File: docs/products/opensearch/howto/opensearch-log-integration.rst [hunk 3]
```diff
@@ -12,23 +12,33 @@ Here are the steps needed to enable logs integration. This allows you to send lo
...
 
+Logs integration configuration
+------------------------------
+There are two parameters that you can adjust when integrating logs to your OpenSearch service:
+
+* ``index prefix``, specifies the prefix part.
+* ``index retention limit``, number of days to preserve the daily indexes.
+
+.. warning::
+    When the ``index retention limit`` is reached, the daily indexes are deleted. The count starts from the day that those indexes were sent.
```
Comment (anelook) [inline docs/products/opensearch/howto/opensearch-log-integration.rst:36]: I'm not sure I understand what means "The count starts from the day that those indexes were sent." , is  "were sent" means "were created"? Would be helpful to be more specific and clear here

**Review by anelook — CHANGES_REQUESTED** [review]
Nice addition to the material! I left a couple of minor comments.

File: docs/products/opensearch/howto/opensearch-log-integration.rst (hunk 1 above)
Comment (laysauchoa) [reply docs/products/opensearch/howto/opensearch-log-integration.rst]: I can leave as it is

**Comment by laysauchoa** [comment]
I think is more clear now @anelook

**Comment by laysauchoa** [comment]
I have tested and after the ``index retention limit`` is reached, the indexes gets deleted from the OpenSearch service.

File: docs/products/opensearch/howto/opensearch-log-integration.rst (hunk 1 above)
Comment (lornajane) [reply docs/products/opensearch/howto/opensearch-log-integration.rst]: Sorry, I think "log integration" singular as well

File: docs/products/opensearch/howto/opensearch-log-integration.rst [hunk 4]
```diff
@@ -8,27 +8,39 @@ In this article, you will understand how to enable, edit and disable the logs in
...
+
+
 Change logs integration configuration
--------------------------------------
+'''''''''''''''''''''''''''''''''''''
 
-You can change the configuration of the ``index prefix`` and ``retention limit`` after the integration is enabled.
+You can change the configuration of the ``index prefix`` and ``index retention limit`` after the integration is enabled.
 
 1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
```
Comment (lornajane) [inline docs/products/opensearch/howto/opensearch-log-integration.rst:45]: What is this line doing here?

File: docs/products/opensearch/howto/opensearch-log-integration.rst [hunk 5]
```diff
@@ -8,27 +8,39 @@ In this article, you will understand how to enable, edit and disable the logs in
...
-5. Configure your ``index prefix`` and ``retention limit`` parameters, then select **Enable**
+5. Configure your ``index prefix`` and ``index retention limit`` parameters, then select **Enable**
 
 .. note::
-    If you want to effectively disable the index ``retention limit``, you can set it to the maximum value which is 10000 days.
+    If you want to effectively disable the ``index retention limit``, you can set it to the maximum value which is 10000 days.
 
 Your Aiven service is now sending logs to your OpenSearch service which you can explore further.
 
+Logs integration configuration
```
Comment (lornajane) [inline docs/products/opensearch/howto/opensearch-log-integration.rst:28]: I think this should be "Configure log integration", using an active verb, and singular "log"

File: docs/products/opensearch/howto/opensearch-log-integration.rst [hunk 6]
```diff
@@ -8,27 +8,39 @@ In this article, you will understand how to enable, edit and disable the logs in
 Enable logs integration
```
Comment (lornajane) [inline docs/products/opensearch/howto/opensearch-log-integration.rst:8]: Singular "log" to go with "integration"

**Review by lornajane — CHANGES_REQUESTED** [review]
Great post! I added some VERY minor comments - let me know if I should make the changes myself when merging or if you would prefer to pick them up yourself?

**Comment by lornajane** [comment]
@laysauchoa will you be able to revisit this one? There are some changes requested

File: docs/products/opensearch/howto/opensearch-log-integration.rst (hunk 4 above)
Comment (laysauchoa) [reply docs/products/opensearch/howto/opensearch-log-integration.rst:45]: I can remove it but I have added the first step from all of them sections.

File: docs/products/opensearch/howto/opensearch-log-integration.rst (hunk 5 above)
Comment (laysauchoa) [reply docs/products/opensearch/howto/opensearch-log-integration.rst]: done

File: docs/products/opensearch/howto/opensearch-log-integration.rst (hunk 6 above)
Comment (laysauchoa) [reply docs/products/opensearch/howto/opensearch-log-integration.rst]: done

**Comment by laysauchoa** [comment]
ping @lornajane I've addressed some of the comments

**Review by lornajane — APPROVED** [review]
Thanks @laysauchoa , this looks good!

**Pull request merged**